docs/community/tutorials-contest/2022/avax-subnet-customization/README.md

```diff
@@ -0,0 +1,975 @@
+
+### Game.sol
+A simple contract I made for the sake of this tutorial.
+
+Basically, the player calls the `play` function with sending some `LUCK` tokens. The function generates a random number and by 50% chance, the player gets 2x the amount of tokens; otherwise, the tokens are stuck in the contract.
+
+```solidity
+pragma solidity ^0.8.7;
+import "./NativeMinterInterface.sol";
+
+contract Game {
+
+    // native minter is deployed at 0x0200000000000000000000000000000000000001
+    NativeMinterInterface minter = NativeMinterInterface(0x0200000000000000000000000000000000000001);
+
+    function random() internal view returns (uint) {
+        // never use this random number generation in a real project
+        // it's deterministic and easily hackable.
+        return uint(keccak256(abi.encodePacked(block.timestamp)));
+    }
+
+    function play() external payable {
+        uint256 randomNumber = random();
+
+        if(randomNumber % 2 == 0){
+            minter.mintNativeCoin(
+                msg.sender,
+                msg.value * 2
+            );
+        }
+    }
+}
+```
+
+When you're done, it should look like this:
+
+![files](.github/images/files.png)
+
+### Compiling and deploying
+
+After creating the files, we should compile `Game.sol`.
+
+![Compile](.github/images/compiling.png)
+
+Go to the deploy tab, choose injected web3 as enviroment - connect your wallet - then choose Game in the contract dropdown. Then click the deploy button.
+
+![Deploy](.github/images/deploying.png)
+
+Confirm the transaction with your metamask wallet.
+
+![Deployed contracts](.github/images/game_deployed.png)
+
+Yay! We deployed a smart contract in our Subnet! That sounds fantastic!
+
+## Setting Game Contract as a Minter
+
+We're admin in [`contractNativeMinterConfig`](#contractnativeminterconfig).
+Which means we can add new Minters to the network using `NativeMinterInterface`.
+
+We already created `NativeMinterInterface.sol`
+
+Go to remix and compile `NativeMinterInterface.sol` by following the same steps as `Game.sol`.
+
+Go to the deploy tab in remix, and choose NativeMinterInterface in the contract dropdown. Then type in the address `0x0200000000000000000000000000000000000001` to `Load contract from Address` then click `At Address`
+
+![Interface steps](.github/images/interface.png)
+
+After clicking the button, you should see the `NATIVEMINTERINTERFACE AT 0X02..` under the `Deployed Contracts` section.
+
+We will call the `setEnabled` function of `NativeMinter` contract with the address of `Game` contract to set the `Game` contract as a minter.
+
+![Set minter](.github/images/setenabled.png)
+
+Confirm the transaction with your metamask wallet. We can call this function because we've added our address as an admin in the `contractNativeMinterConfig` section.
+
+Now the `Game.sol` contract is a minter and can mint native tokens!
+
+## Play the Game
+
+In the deploy tab: 
+- Set the value to 100 Ether or any other amount you want to play with.
```

Comment:
Per [Markdownlint](https://github.com/DavidAnson/markdownlint) Lists should be surrounded by blank lines.

> MD032/blanks-around-lists: Lists should be surrounded by blank lines

[More info: MD032](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md032---lists-should-be-surrounded-by-blank-lines)